ci/scripts/target-location.sh

```diff
@@ -11,12 +11,14 @@ readonly REPO_TOP
 
 verbose='false'
 print_usage() {
-  printf "Usage: $0 [-v] <bazel target label>"
+  printf "Usage: $0 [-v] [-b bazel option 1] [-b bazel option 2] <bazel target label>"
```

Comment:
Yeah, this is awkward but I couldn't think of a better way for passing an option with spaces and wanted to push this as quickly as possible -- hopefully we'll again use the cached strategy with the broken rom but this time CI will pass. PLMK if this can be handled more ergonomically.

Reply:
We could choose to split args with the venerable `--` option. In other words, upon seeing `--`, shift that option out, and break out of the loop, leaving the rest of the arguments for the subprocess.

Reply:
Thanks @a-will! I was about to push an update that implements this with a `for` loop but realized that the original script already passed `"$@"` to `bazelisk.sh outquery`. So, I just updated the usage string PTAL.